server/server.go

```diff
@@ -195,22 +325,74 @@ func (c *Server) setupCommandQueue(logger log.Logger) error {
 
 func (c *Server) setupBolt() error {
 	dbPath := filepath.Join(c.ConfigPath, "micromdm.db")
+
 	db, err := bolt.Open(dbPath, 0644, &bolt.Options{Timeout: time.Second})
 	if err != nil {
 		return errors.Wrap(err, "opening boltdb")
 	}
+
 	c.DB = db
 
 	return nil
 }
 
+func (c *Server) setupRdbms() error {
+	
+	if (c.Rdbms == Rdbms(boltdb) || c.RdbmsUsername == "" || c.RdbmsPassword == "" || c.RdbmsHost == "" || c.RdbmsPort == "" || c.RdbmsDatabase == "") {
+		return nil
+	}
+
+	if c.Rdbms == mysql {
+		db, err := dbutil.OpenDBX(
+			c.Rdbms.String(),
+			c.RdbmsUsername+":"+c.RdbmsPassword+"@tcp("+c.RdbmsHost+":"+c.RdbmsPort+")/"+c.RdbmsDatabase+"?parseTime=true",
+			dbutil.WithLogger(log.NewNopLogger()),
+			dbutil.WithMaxAttempts(1),
+		)
+		if err != nil {
+			return errors.Wrap(err, "opening mysql")
+		}
+		
+		c.MysqlDB = db
+		// Set the number of open and idle connection to a maximum total of 2.
+		db.SetMaxOpenConns(1)
+		db.SetMaxIdleConns(1)
+		
+	} else if c.Rdbms == postgres {
+		db, err := dbutil.OpenDBX(
+			"postgres",
+			//"host=localhost port=5432 user=micromdm dbname=micromdm_test password=micromdm sslmode=disable",
+			"host="+c.RdbmsHost+" port="+c.RdbmsPort+" user="+c.RdbmsUsername+" dbname="+c.RdbmsDatabase+" password="+c.RdbmsPassword+" sslmode=disable",
```

Comment:
`sslmode` should be exposed so it can be set to _require_ or _disabled_ as needed.